omero/sysadmins/unix/install-web/web-deployment.rst

```diff
@@ -102,20 +98,29 @@ configuration redirect the output of the following command into a file:
 .. literalinclude:: nginx-omero.conf
 
 
-OMERO.web requires ``body_in_file_only`` adjusted in your default nginx
```

Comment:
I meant I can't find any mention of what it should be set to or why it's needed in our code or docs, nor any mention of it in our infrastructure repos.